seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/util/DorisCatalogUtil.java

```diff
@@ -110,21 +111,27 @@ public static String getCreateTableStatement(
         String template = createTableTemplate;
         TableSchema tableSchema = catalogTable.getTableSchema();
 
-        String primaryKey = "";
+        String primaryKey = StringUtils.EMPTY;
```

Comment:
I think the old code style more clear.
```suggestion
        String primaryKey = "";
```